src/indra_cogex/client/queries.py

```diff
@@ -415,3 +424,273 @@ def isa_or_partof(
     """
     term_parents = get_ontology_parent_terms(client, term)
     return any(parent == parent_term.grounding() for parent_term in term_parents)
+
+
+# MESH / PMID
+
+
+def get_pmids_for_mesh(
+    client: Neo4jClient, mesh: Tuple[str, str], include_child_terms: bool = True
+) -> Iterable[str]:
+    """Return the PubMed IDs for the given MESH term.
+
+    Parameters
+    ----------
+    client :
+        The Neo4j client.
+    mesh :
+        The MESH term to query.
+    include_child_terms :
+        If True, also match against the child MESH terms of the given MESH
+        term.
+
+    Returns
+    -------
+    :
+        The PubMed IDs for the given MESH term and, optionally, its child terms.
+    """
+    if mesh[0].lower() != "mesh":
+        raise ValueError(f"Expected mesh term, got {':'.join(mesh)}")
+
+    # NOTE: we could use get_ontology_child_terms() here, but it's ~20 times
+    # slower for this specific query, so we do an optimized query that is
+    # basically equivalent instead.
+    if include_child_terms:
+        child_query = (
+            """
+            MATCH (c:BioEntity)-[:isa|partf*1..]->(p:BioEntity)
```

Comment:
Typo in part of?

Reply:
we don't need partof here

Reply:
Yup, will fix and check query

Reply:
Although, I just realized, isn't the mesh hierarchy strictly `isa` relationships? Then I shouldn't use `partof` in the first place.

Reply:
Yes:
https://github.com/sorgerlab/indra/blob/master/indra/ontology/bio/ontology.py#L368-L370

Reply:
Resolved in https://github.com/bgyori/indra_cogex/pull/50/commits/e9c784e5ae34e55686b5e77e273bfc47d474c09e

Reply:
(oh, sorry, your comment wasn't visible until now @bgyori, the page must have been outdated when it woke up the laptop after coming in)